views/languages.yaml

```diff
@@ -5,6 +5,9 @@ path:
 template: Language
 context:
   dynamic:
-    delimiter_type_comment: session.query(EnumDelimiterType).filter(EnumDelimiterType.name == 'comment').one()
-    delimiter_type_string: session.query(EnumDelimiterType).filter(EnumDelimiterType.name == 'string').one()
-    delimiter_type_block: session.query(EnumDelimiterType).filter(EnumDelimiterType.name == 'block').one()
+    delimiter_type_comment: session.query(EnumDelimiterType).filter
+(EnumDelimiterType.name == 'comment').one()
+    delimiter_type_string: session.query(EnumDelimiterType).filter
```

Comment:
syntax error: could not find expected ':'

*Origin: YAMLLintBear, Section: `yaml`.*